website/docs/en/config/entry.mdx

```diff
@@ -196,16 +196,30 @@ The default value can be affected by different [target](/config/target):
 <ApiMeta addedVersion={'1.0.0-beta.1'} />
 
 <PropertyType
-  type="string | null"
+  type="string | null | undefined"
   defaultValueList={[{ defaultValue: 'undefined' }]}
 />
 
-The layer of this entry, make the corresponding configuration take effect through layer matching in SplitChunks, Rules, Stats, and Externals.
```

Comment:
@ahabhgk Does Rspack support layers in split chunks and stats now?

Reply:
Get, I will submit another PR to improve the documentation for split chunks and stats.